packages/amplify-category-auth/src/commands/auth.ts

```diff
@@ -9,16 +9,6 @@ type AuthCommandType = {
   description: string;
 };
 
-/**
- * Push Auth resources to the cloud
- * @param context amplify cli context
- * @returns push command response
- */
-export const authPush = async (context: $TSContext): Promise<$TSAny> => {
```

Comment:
export not needed as we could import directly in packages/amplify-category-auth/src/index.js